ccm_web/client/src/components/CreateSectionWidget.tsx

```diff
@@ -53,6 +53,9 @@ function CreateSectionWidget (props: CreateSectionWidgetProps): JSX.Element {
           .then(newSections => {
             props.onSectionCreated(newSections[0])
             setNewSectionName('')
+            enqueueSnackbar('Section name has been created', {
+              variant: 'success'
+            })
```

Comment:
```suggestion
            enqueueSnackbar('Section was created!', { variant: 'success' })
```

Reply:
@janelilr are you ok with this message change to `Section was created!` ?